spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalDeployerAutoConfiguration.java

```diff
@@ -35,13 +35,15 @@
 @AutoConfigureOrder(Ordered.HIGHEST_PRECEDENCE)
 public class LocalDeployerAutoConfiguration {
 
-	@Bean
+	public static final String DEPLOYER = "Local";
+
+	@Bean(DEPLOYER + "AppDeployer")
 	@ConditionalOnMissingBean(AppDeployer.class)
 	public AppDeployer appDeployer(LocalDeployerProperties properties) {
```

Comment:
I thought about it. Then I thought it would be good to have the `prefix` as a property so that it can be used outside (for instance when choosing the specific deployer based on the derived name)